stdlib/public/Concurrency/SuspendingClock.swift

```diff
@@ -0,0 +1,137 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2021 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+import Swift
+
+@available(SwiftStdlib 9999, *)
+public struct SuspendingClock {
+  public struct Instant: Codable, Sendable {
```

Comment:
Do we want to provide a `CustomStringConvertible` conformance for these instant types?

We definitely don't want to print them in a calendrical format, but it might be a good idea to customize the default reflection-based output. (Which looks something like `Instant(_value: 23.456 seconds)`. Would something like `T+23.456s` be less annoying to read in the long term?)

Reply:
probably; good point